application/routes/default.php

```diff
@@ -215,6 +203,20 @@
 	));
 
 /**
+ * Forms API SubRoute
+ */
+Route::set('forms', $apiBase . 'forms(/<form_id>)/<controller>(/<id>)',
```

Comment:
nm. I see the change. I'm not 100% why this moved down the file though? vs just making the 1 line change.